lib/modules/apostrophe-assets/index.js

```diff
@@ -192,6 +192,10 @@ module.exports = {
     // For generating unique keys cheaply
     self.ordinal = 0;
 
+    if (self.options.jQuery !== 3) {
+      self.apos.utils.warnDev('\n\n⚠️  Apostrophe is not configured to use jQuery v3.⚠️\nThis creates a security vulnerability. Please update apostrophe-assets configuration. For more information: https://apos.dev/jquery-3\n\n');
```

Comment:
btw, I like this pattern of creating vertical space, then having a "headline" using another line break. I'm trying to have it pop without people going blind to the warning.

Reply:
agreed.